### PR TITLE
Add all contributors to readme

### PR DESCRIPTION
**Pull request by ArchanRD**
Closes issue #72

### CODE REVIEW

**Comment by ArchanRD** [comment]
@muruga21 Please review the PR.

File: README.md [hunk 1]
```diff
@@ -88,3 +88,16 @@ Make your changes.
...
+
+---
+
+## Our Awesome Contributors 🚀
+
+- Thank you for your valuable contribution to this project. We truly appreciate your skills and dedication. Keep up the great work!
+- Make sure you show some love by giving ⭐ to our repository.
+
+<div align="left">
+  <a href="https://github.com/AlfiyaSiddique/TastyTrails">
```
Comment (muruga21) [inline README.md:100]: bro.. why did you add your repo link here ??

File: README.md [hunk 2]
```diff
@@ -88,3 +88,16 @@ Make your changes.
...
+    <img src="https://contrib.rocks/image?repo=AlfiyaSiddique/TastyTrails&&max=10" />
```
Comment (muruga21) [inline README.md:101]: what is this like dude..

image

i think these are not the contributors from this repo..

**Review by muruga21 — CHANGES_REQUESTED** [review]

**Comment by muruga21** [comment]
@ArchanRD and you have mentioned the contributors list will be dynamic right ?, im glad you came up with this idea dude.. please try to complete the mentioned changes dude

File: README.md (hunk 1 above)
Comment (ArchanRD) [reply README.md:100]: Ohh, I am sorry. I was testing it out and forgot to change the repo link!

**Review by muruga21 — APPROVED** [review]

**Comment by ArchanRD** [comment]
Hi @muruga21 If the checks are accepted, can you please merge this PR?

**Comment by muruga21** [comment]
@ArchanRD Thanks for your contribution
